### PR TITLE
Update the kokoro build version in gcloud command

### PR DESCRIPTION
**Pull request by AspirinSJL**

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by jtattermusch** [comment]
@yihuazhang   I'm 99% sure that this PR doesn't break ALTS interop tests - the test probably started failing on master for a different reason  and it's a pure coincidence that this PR was merged around that time. Did we check if there were any changes on the Go side by the time the ALTS tests started failing?

**Comment by yihuazhang** [comment]
@jtattermusch, 

I just re-triggered the tests, and the broken tests dropped from 95 to 2. The problem seems to be at ALTS Go stack, and we are investigating it now.

File: tools/gce/create_windows_debug_worker.sh [hunk 1]
```diff
@@ -49,7 +49,8 @@ gcloud compute instances create "$INSTANCE_NAME" \
     --zone "$ZONE" \
     --machine-type "$MACHINE_TYPE" \
     --image-project google.com:kokoro \
-    --image kokoro-win7build-v9-prod-debug \
+    # The version might need updating.
```
Comment (dfawley) [inline tools/gce/create_windows_debug_worker.sh]: I don't think this works.  This line terminates the command above, and the next command is "--image kokoro-win7build........"

Comment (AspirinSJL) [reply tools/gce/create_windows_debug_worker.sh]: Yeah.. Good catch!

**Pull request merged**